lib/crowbar/init/application.rb

```diff
@@ -31,26 +31,30 @@ class Application < Sinatra::Base
         set :environment, ENV["RAILS_ENV"]
       end
 
+      if settings.environment == :development
+        set :logpath, "#{settings.root}/log"
+      else
+        set :logpath, "/var/log/crowbar"
+      end
+
+      configure do
+        ::Logger.class_eval { alias :write :'<<' }
```

Comment:
Why is it `<<` instead of `>>`? I don't fully understand how this works here, but reading this from a bash point of view this seems strange (but then again, it's Ruby :wink:).